src/language-html/printer-html.js

```diff
@@ -1102,7 +1110,7 @@ function printEmbeddedAttributeValue(node, originalTextToDoc, options) {
     /**
      *     v-if="jsExpression"
      */
-    const jsExpressionBindingPatterns = ["^v-", "^#"];
+    const jsExpressionBindingPatterns = ["^v-"];
```

Comment:
The `#default="..."` is added in #9609, and it should not be here.

Reply:
Ok, thanks for mentioning.